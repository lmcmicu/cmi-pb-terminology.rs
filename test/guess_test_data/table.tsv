table	path	description	type
column	test/guess_test_data/column.tsv	Columns for all of the tables.	column
datatype	test/guess_test_data/datatype.tsv	Datatypes for all of the columns	datatype
rule	test/guess_test_data/rule.tsv	More complex "when" rules	rule
table	test/guess_test_data/table.tsv	All of the user-editable tables in this project.	table
table1	test/guess_test_data/table1.tsv	The first data table	
