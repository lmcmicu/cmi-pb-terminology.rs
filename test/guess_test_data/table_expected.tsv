table	path	description	type
column	test/guess_test_data/column_expected.tsv	Columns for all of the tables.	column
datatype	test/guess_test_data/datatype.tsv	Datatypes for all of the columns	datatype
rule	test/guess_test_data/rule.tsv	More complex "when" rules	rule
table	test/guess_test_data/table_expected.tsv	All of the user-editable tables in this project.	table
table1	test/guess_test_data/table1.tsv	The first data table	
table2	test/guess_test_data/ontology/table2.tsv	The second data table	
table3	test/guess_test_data/ontology/table3.tsv	The third data table	
table4	test/guess_test_data/ontology/table4.tsv	The fourth data table	
table5	test/guess_test_data/ontology/table5.tsv	The fifth data table	
table6	test/guess_test_data/ontology/table6.tsv	The sixth data table (like table2 but all numeric)	
